.circleci/config.yml

```diff
@@ -69,6 +70,57 @@ commands:
           name: print installed packages
           command: pip freeze --all
 
+  dependencies-windows:
+    # You must still manually update the Docker image if any
```

Comment:
are we using Docker with windows?

Reply:
No, we are using the Windows executor provided by CircleCI orbs. I'll remove that comment.